src/components/oscal-utils/OSCALRestUtils.js

```diff
@@ -0,0 +1,214 @@
+/**
+ * enum describing the available REST methods
+ */
+ export const restMethods = {
+  DELETE: "DELETE",
+  GET: "GET",
+  PATCH: "PATCH",
+  POST: "POST",
+  PUT: "PUT",
+};
+
+export const oscalObjectTypes = {
```

Comment:
It's debatable whether the `name`, `defaultUrl`, and `defaultUuid` really belong here.  We may want to pull them out and into another enum-like object in a different file.

Reply:
I'd say it makes sense to pull them into a separate file, but only when we break `OSCALRestUtils` down into more files to separate out different aspects of the REST requests or OSCAL types and generic OSCAL utils (which could also contain the `deepClone` function). We are not at that point right now, but should note that they are not similar enough to other functions/enums in this file to keep them together in the same file long-term.